testdata/goldens/java/com.google.cloud.speech.v1.SpeechClient.html

```diff
@@ -18,7 +18,7 @@ <h1 class="page-title">Class SpeechClient
   <div class="markdown level0 summary"><p>Service Description: Service that implements Google Cloud Speech API.</p>
 <p> <p>This class provides the ability to make remote calls to the backing service through method
  calls that map to API methods. Sample code to get started:</p>
- <pre><code class="pretty-print">
+ <pre><code class="prettyprint lang-java">
```

Comment:
Obviously not for this PR, but the `prettyprint` should be on the `pre` to make sure click to copy works. See #271.

Reply:
Yeah this is a bug in the doclet. Easy fix, updated now.